### PR TITLE
Adds additional items to the storage of many coats and oversuits.

### PR DESCRIPTION
**Pull request by harmonyn**
Title says it all. Quality of life, logic, etc. Please suggest more changes if you like.

List of added items so far:

LABCOATS:
- Soap (for medical cleanup)

FIRESUITS:
- Crowbars (firelock opening)

BIOSUITS:
- Hypospray, dropper, syringe (for MDs, last can also be useful for xenobio)

WALLETS:
- Lipstick
- Syringes (droppers are already allowed)

MINING HARDSUITS:
- Kinetic accelerator
- Resonator
- Mining scanners

BOTANIST APRONS:
- Beakers (for mutagen)

PIRATE COATS/SPACESUITS:
- Energy cutlass
- Eyepatch
- Bottle of rum

XENO SUIT:
- Toy facehugger (honk)

### CODE REVIEW

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -322,7 +323,7 @@
 	item_state = "xenos_helm"
 	body_parts_covered = CHEST|GROIN|LEGS|ARMS|HANDS
 	flags_inv = HIDEGLOVES|HIDESHOES|HIDEJUMPSUIT
-
+	allowed = list(/obj/item/clothing/mask/facehugger/toy)
```
Comment (Anonus) [inline code/modules/clothing/suits/miscellaneous.dm]: make it all facehuggers, it'll be more amusing

Comment (bgobandit) [inline code/modules/clothing/suits/miscellaneous.dm]: I was considering that but didn't know it would mess with the facehugging code

Comment (Anonus) [inline code/modules/clothing/suits/miscellaneous.dm]: You can't even pick up facehuggers under most circumstances anyway.

**Comment by WJohn** [comment]
:+1:

**Comment by PKPenguin321** [comment]
labcoats carrying soap just sounds powergamey to me
what legitimate reason could you ever have for carrying soap in a lab coat

**Comment by bgobandit** [comment]
to clean up the massive amounts of blood, dried gibs and vomit that tend to accumulate in the medbay? medbay already starts with soap to begin with

File: code/modules/clothing/suits/miscellaneous.dm (hunk 1 above)
Comment (GunHog) [inline code/modules/clothing/suits/miscellaneous.dm]: Well, you can take them off your face if you recover from the stun before they finish tonguing you.

Comment (Anonus) [inline code/modules/clothing/suits/miscellaneous.dm]: At that point I don't think it matters if you can put them in your suit that much.

**Comment by Cheridan** [comment]
wtf kind of huge wallet are you carrying around that would fit syringes and lipstick

**Comment by duncathan** [comment]
Syringes probably not, but lipstick makes sense

**Comment by Lumipharon** [comment]
Man if you've ever worn a lab coat before, you'll know those fuckers have huge ass pockets.

Honestly you should be able to fit anything in there that you can fit in your actual pockets.

**Comment by bgobandit** [comment]
wallets are fucking big man

**Comment by Dorsisdwarf** [comment]
Ready for review?

**Comment by bgobandit** [comment]
yeah

**Comment by Cheridan** [comment]
i call shenanigans

post pics of your wallet

**Comment by bgobandit** [comment]
hey man it's not your fault you're too broke to need a regular size wallet :^)

**Comment by Bawhoppen** [comment]
how can miner hardsuits still not carry mining scanners or KAs (and probably other things) in their suit storage?

**Comment by bgobandit** [comment]
GOOD QUESTION

adding those

**Comment by bgobandit** [comment]
added

**Comment by Fox-McCloud** [comment]
@Lumipharon 

I know Bay turned them into full on storage--they have two storage slots in them---for those two huge pockets most have.

**Comment by SweaterKittens** [comment]
Don't forget EMT jackets in all of this, they should be able to hold the same sort of things as Lab Coats for the aspiring Field Medic.

**Pull request merged**